### PR TITLE
Feat/deploy new guides port 24

### PR DESCRIPTION
**Pull request by 3avi**
# New Guides Navbar/Sidebar - #76

### CODE REVIEW

File: website/docs/guides/data-modeling/composites.mdx [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+await composite.startIndexingOn(ceramic)
+```
+
+  </TabItem>
+</Tabs>
+
+This will also automatically add all models contained in the composite to the [Model Catalog](./model-catalog). 
+
+### Compiling composites
+After deploying your composite, compile it so you can start perform [data Interactions](notion.so guide) using the [ComposeDB Client](notion.so app setup)
```
Comment (PaulLeCam) [inline website/docs/guides/data-modeling/composites.mdx:140]: Notion links

File: website/docs/guides/data-modeling/composites.mdx [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+const sourceComposites = await Promise.all(loadSources)
+const mergedComposite = Composite.from(sourceComposites)
+
+await writeEncodedComposite(mergedComposite, 'merged-composite.json')
+```
+
+:::caution
+***Note:*** To run the code above, you need a Ceramic JS HTTP client library installed on your machine to connect your app to a Ceramic node. Ceramic JS HTTP client can be installed by running the following command:
+
+`pnpm install @ceramicnetwork/http-client`[](https://composedb.js.org/docs/0.3.x/api/commands/cli.graphql#composedb-graphqlschema)
```
Comment (PaulLeCam) [inline website/docs/guides/data-modeling/composites.mdx:193]: v0.3 link

File: website/docs/guides/data-modeling/composites.mdx [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+This will create a file called `new-composite.json` with your profile model in it. From here you need to deploy the composite to your node, then [compile](#compiling-composites) the composite to start using it. 
+
+### Inspecting composites
+If you want to check what models are included in a specific composite, follow the steps below:
+
+1. Compile the composite:
+`composedb composite:compile my-first-composite.json runtime-composite.json`
+
+2. View the GraphQL schema of the composite:
+`composedb graphql:schema runtime-composite.json --output=schema.graphql`[](https://composedb.js.org/docs/0.3.x/api/commands/cli.graphql#composedb-graphqlschema)
```
Comment (PaulLeCam) [inline website/docs/guides/data-modeling/composites.mdx:244]: v0.3 link, also seems to be an empty link (`[]`)

File: website/docs/guides/data-modeling/composites.mdx [hunk 4]
```diff
@@ -0,0 +1,296 @@
...
+
+From here you need to [deploy](#deploying-composites) the composite to your node, then [compile](#compiling-composites) the composite to start using it. When interacting with the models inside your app, you can refer to them using their human-readable aliases rather than their streamIDs. 
+
+## Next Steps
+- [Application Setup](https://www.notion.so/Application-Setup-4ba61ae9757f4fbf9769aff4e95d1b93) to set up your server and configure your app
+- [Data Interactions](https://www.notion.so/Data-Interactions-249a50eba41341e8954511175f468fa2) to interact with data using your composite
+
+## Related Guides
+- [Model Catalog](https://www.notion.so/Model-Catalog-e2eddcca410f4a19b533fdccd7c5c6a0)
+- [Writing Models](https://www.notion.so/Writing-Models-e829e0e595574a0d90987f9acdc011f0)
```
Comment (PaulLeCam) [inline website/docs/guides/data-modeling/composites.mdx:296]: Links to Notion pages

File: website/docs/guides/data-modeling/introduction-to-modeling.mdx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+}
+```
+
+Where:
+
+- `displayName` is a field
+- `String!` is a scalar that defines `displayName` is a required (`!`) string
+- `@string` is a directive that sets validation rules for the scalar, in our case min and max length
+
+This is a very basic schema, but your schemas can contain more than one field and include various relations - see [Schemas](https://www.notion.so/Schemas-0dd714405557435d8968e0ac23d5c4cb).
```
Comment (PaulLeCam) [inline website/docs/guides/data-modeling/introduction-to-modeling.mdx:46]: Notion

File: website/docs/guides/data-modeling/introduction-to-modeling.mdx [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+## Next Steps
+To use your new model in your application, you will need to create a [Composites](https://www.notion.so/Composites-d92f5986e2594ada841586efb21bb5a4).
```
Comment (PaulLeCam) [inline website/docs/guides/data-modeling/introduction-to-modeling.mdx:49]: Notion

File: website/docs/guides/data-modeling/introduction-to-modeling.mdx [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+## Related Guides
+- Dive deeper into GraphQL [Schemas](https://www.notion.so/Schemas-0dd714405557435d8968e0ac23d5c4cb)
+- Learn how to add [Relations](https://www.notion.so/Relations-60f78e9672d04ef29a1002a38d7706b7) to your models
```
Comment (PaulLeCam) [inline website/docs/guides/data-modeling/introduction-to-modeling.mdx:53]: Notion

**Review by PaulLeCam — CHANGES_REQUESTED** [review]
I haven't gone through all the pages but there are already many links pointing to Notion pages so can you let me know once these are replaced please?

**Comment by michaelsena** [comment]
@3avi can you resolve comments if you've addressed them?

File: website/docs/api/commands/cli.composite.md [hunk 8]
```diff
@@ -23,11 +22,10 @@ creation and interactions with
 ### `composedb composite:from-model`
 
 Create an encoded composite definition from a list of already existing model
-stream ids (usually found by
-[Composites Discovery](../../guides/using-composites/discovery.mdx))
+stream ids (usually found within [Model Catalog](../../guides/data-modeling/model-catalog.mdx)
 
 You can find a detailed guide on using an existing model to create your
-composite [here](../../create-your-composite.mdx)
+composite [here](../../guides/data-modeling/composites)
```
Comment (PaulLeCam) [inline website/docs/api/commands/cli.composite.md:28]: Missing extension

File: website/docs/api/commands/cli.composite.md [hunk 9]
```diff
@@ -44,11 +42,9 @@ OPTIONS
 
 ### `composedb composite:create`
 
-Create an encoded composite definition from GraphQL
-[Composite Schema](../../guides/creating-composites/overview.mdx#composite-schema)
+Create an encoded composite definition from GraphQL [Composite Schema](../../guides/data-modeling/composites.mdx)
 
-<!-- You can find a detailed guide on the creation of Composites [here](../.././create-your-composite)
-replace with data modelling composites section -->
+You can find a detailed guide on the creation of Composites [here](../../guides/data-modeling/composites)
```
Comment (PaulLeCam) [inline website/docs/api/commands/cli.composite.md:47]:
```suggestion
You can find a detailed guide on the creation of Composites [here](../../guides/data-modeling/composites.mdx)
```

File: website/docs/api/commands/cli.document.md [hunk 10]
```diff
@@ -18,7 +18,7 @@ The group of [CLI](../modules/cli.md) `document:*` commands enables the creation
 
 Create a document with given content
 
-You can use this command to test how document creation works, but in your live app you will use ComposeDB GraphQL Mutations instead. <!-- replace with guide > [](../../guides/interacting/mutations.mdx) -->
+You can use this command to test how document creation works, but in your live app you will use ComposeDB GraphQL [Mutations](../../guides/data-interactions/mutations) instead.
```
Comment (PaulLeCam) [inline website/docs/api/commands/cli.document.md:21]: Missing file extension

File: website/docs/api/commands/cli.document.md [hunk 11]
```diff
@@ -37,7 +37,7 @@ OPTIONS
 
 Replace content in a document with a given streamID
 
-You can use this command to test how document updates work, but in your live app you will use ComposeDB GraphQL Mutations. <!-- [] (../../guides/interacting/mutations.mdx) -->
+You can use this command to test how document updates work, but in your live app you will use ComposeDB GraphQL [Mutations](../../guides/data-interactions/mutations).
```
Comment (PaulLeCam) [inline website/docs/api/commands/cli.document.md:40]: Missing file extension

File: website/docs/api/commands/cli.document.md [hunk 12]
```diff
@@ -56,7 +56,7 @@ OPTIONS
 
 Load and display the contents of a document with a given ID
 
-You can use this command to manually check the contents of a given document, but in your DApp you will use ComposeDB GraphQL Queries. <!-- [](../../guides/interacting/queries.md) -->
+You can use this command to manually check the contents of a given document, but in your DApp you will use ComposeDB GraphQL [Queries](../../guides/data-interactions/queries).
```
Comment (PaulLeCam) [inline website/docs/api/commands/cli.document.md:59]: Missing file extension

File: website/docs/api/commands/cli.graphql.md [hunk 13]
```diff
@@ -4,7 +4,7 @@ title: "CLI: graphql:* commands"
 custom_edit_url: null
 ---
 
-The group of [CLI](../modules/cli.md) `graphql:*` commands makes it possible to generate GraphQL Schemas from [runtime definitions](./cli.composite.md#composedb-compositecompile) of your [Composites](../../data-modeling-concepts.mdx#composites) and run a local GraphQL HTTP server
+The group of [CLI](../modules/cli.md) `graphql:*` commands makes it possible to generate GraphQL Schemas from compiled [Composites](../../guides/data-modeling/composites) and run a local GraphQL HTTP server
```
Comment (PaulLeCam) [inline website/docs/api/commands/cli.graphql.md:7]:
```suggestion
The group of [CLI](../modules/cli.md) `graphql:*` commands makes it possible to generate GraphQL Schemas from compiled [Composites](../../guides/data-modeling/composites.mdx) and run a local GraphQL HTTP server
```

File: website/docs/api/commands/cli.graphql.md [hunk 14]
```diff
@@ -18,7 +18,8 @@ The group of [CLI](../modules/cli.md) `graphql:*` commands makes it possible to
 Load the runtime graphql schema for the composite with given runtime definition
 
 You can use this command to see the ComposeDB GraphQL Execution Schema for your composite.
-You will be using this schema in your DApp to perform Queries and Mutations. <!-- [Queries](../../guides/interacting/mutations.mdx) and [Mutations](../../guides/interacting/mutations.mdx) -->
+
+You will be using this schema in your app to perform [Queries](../../guides/data-interactions/queries) and [Mutations](../../guides/data-interactions/mutations).
```
Comment (PaulLeCam) [inline website/docs/api/commands/cli.graphql.md:22]: Missing file extension

File: website/docs/api/commands/cli.model.md [hunk 15]
```diff
@@ -19,7 +19,7 @@ The group of [CLI](../modules/cli.md) `model:*` commands enables discovery of [M
 
 Display a paginated list of models indexed on the connected ceramic node, which needs to be indexing model streams.
 
-This currently is the main entry path to [Composites Discovery](../../data-modeling-concepts.mdx#composites) <!-- replace with dedicated composites discovery page -->
+This currently is the main entry path to the [Model Catalog](../../guides/data-modeling/model-catalog)
```
Comment (PaulLeCam) [inline website/docs/api/commands/cli.model.md:22]: Missing file extension

File: website/docs/api/commands/cli.model.md [hunk 16]
```diff
@@ -34,7 +34,7 @@ OPTIONS
 
 Load the contents of a model stream with a given ID.
 
-You should use this command to check if an existing model found through [Composites Discovery](../../data-modeling-concepts.mdx#composites) has the fields you need.
+You should use this command to check if an existing model found through [Model Catalog](../../guides/data-modeling/model-catalog) has the fields you need.
```
Comment (PaulLeCam) [inline website/docs/api/commands/cli.model.md:37]: Missing file extension

File: website/docs/core-concepts.mdx [hunk 17]
```diff
@@ -1,13 +1,14 @@
 # Core Concepts
 Learn about the Core Concepts for ComposeDB:
 
-**[Graph](./graph-structure.mdx)**
-ComposeDB provides a graph structure for interacting with data on Ceramic, with two types of nodes: accounts & documents, and edges between nodes representing relations.
+## [Graph](./graph-structure.mdx)
+ComposeDB provides a graph structure for interacting with data on Ceramic, with two types of nodes [accounts](./graph-structure#accounts) & [documents](./graph-structure#documents), and edges between nodes representing relations. [**→**](./graph-structure.mdx)
```
Comment (PaulLeCam) [inline website/docs/core-concepts.mdx:5]: Missing file extension

**Review by 3avi — COMMENTED** [review]
fixing links

**Review by PaulLeCam — CHANGES_REQUESTED** [review]
I haven't been through all the pages but so many links are missing the file extensions already

File: website/docs/api/commands/cli.graphql.md [hunk 18]
```diff
@@ -18,7 +18,8 @@ The group of [CLI](../modules/cli.md) `graphql:*` commands makes it possible to
 Load the runtime graphql schema for the composite with given runtime definition
 
 You can use this command to see the ComposeDB GraphQL Execution Schema for your composite.
-You will be using this schema in your DApp to perform Queries and Mutations. <!-- [Queries](../../guides/interacting/mutations.mdx) and [Mutations](../../guides/interacting/mutations.mdx) -->
+
+You will be using this schema in your app to perform [Queries](../../guides/data-interactions/queries) and [Mutations](../../guides/data-interactions/mutations.mdx).
```
Comment (PaulLeCam) [inline website/docs/api/commands/cli.graphql.md:22]: Missing file extension

File: website/docs/graph-structure.mdx [hunk 19]
```diff
@@ -7,20 +7,16 @@ ComposeDB is is designed similar to a property graph database.
...
-
-<!-- [Documents](../0.3.x/guides/concepts-overview#documents) are [Ceramic streams](https://developers.ceramic.network/learn/advanced/overview/#streams) that store structured data as defined by a [model](../0.3.x/guides/concepts-overview#models). -->
+Documents are [Ceramic streams](https://developers.ceramic.network/learn/advanced/overview/#streams) that store structured data as defined by a [model](./guides/data-modeling/introduction-to-modeling).
 
 Ceramic nodes can be configured to index specific models, storing all the documents using a model in a local database in order to provide fast access and query capabilities.
 
-Using [GraphQL](https://graphql.org/), the ComposeDB client allows to query documents indexed by a Ceramic node, as well as creating new documents and updating existing ones when mutations are enabled. <!-- [mutations are enabled](../0.3.x/guides/interacting/mutations). -->
+Using [GraphQL](https://graphql.org/), the ComposeDB client allows to query documents indexed by a Ceramic node, as well as creating new documents and updating existing ones when [mutations](./guides/data-interactions/mutations) are enabled.
```
Comment (PaulLeCam) [inline website/docs/graph-structure.mdx:22]: Missing file extension

File: website/docs/guides/composedb-client/composedb-client.mdx [hunk 20]
```diff
@@ -0,0 +1,12 @@
+# ComposeDB Client
+Connect your app to a ComposeDB server
+
+## Connect your application
+Interact with ComposeDB using JavaScript, TypeScript, or React
+ 
+**[JavaScript Client →](./../guides/composedb-client/javascript-client)**
```
Comment (PaulLeCam) [inline website/docs/guides/composedb-client/composedb-client.mdx:7]: Missing file extension

File: website/docs/guides/composedb-client/composedb-client.mdx [hunk 21]
```diff
@@ -0,0 +1,12 @@
...
+
+## Authenticate users
+Enable user interactions, including data mutations 
+
+**[Authenticate Users →](./../guides/composedb-client/authenticate-users)**
```
Comment (PaulLeCam) [inline website/docs/guides/composedb-client/composedb-client.mdx:12]: Missing file extension

File: website/docs/guides/composedb-client/user-sessions.mdx [hunk 22]
```diff
@@ -0,0 +1,248 @@
...
+const ethProvider = // import/get your web3 eth provider
+const addresses = await ethProvider.request({ method: 'eth_requestAccounts' })
+const accountId = await getAccountId(ethProvider, addresses[0])
+const authMethod = await EthereumWebAuth.getAuthMethod(ethProvider, accountId)
+
+const compose = new ComposeClient()
+
+const loadSession = async (authMethod) => {
+	// Check if user session already in storage 
+  const sessionStr = localStorage.getItem('didsession')
```
Comment (PaulLeCam) [inline website/docs/guides/composedb-client/user-sessions.mdx:207]: Could you add warnings about using localstorage as in the DID docs please? This shouldn't be treated as a secure option.

File: website/docs/guides/composedb-client/using-apollo.mdx [hunk 23]
```diff
@@ -0,0 +1,38 @@
+# Using Apollo GraphQL Client
+[Apollo](https://www.apollographql.com/docs/react/api/core/ApolloClient) is a popular GraphQL client for React and other platforms.
+
+## Prerequisites
+- Install the [`composedb`](../../set-up-your-environment) packages
```
Comment (PaulLeCam) [inline website/docs/guides/composedb-client/using-apollo.mdx:5]: Missing file extension

File: website/docs/guides/data-interactions/queries.mdx [hunk 24]
```diff
@@ -0,0 +1,168 @@
+# Queries
+
+Access data stored on the network.
+
+## Prerequisites
+- Before your node can index your models, you need to deploy your composite.
+- Before your client can perform queries on those models, you need to [compile your composite](../data-modeling/composites). [`ComposeClient`](../../api/classes/client.ComposeClient) automatically generates a GraphQL Schema from your compiled composite
```
Comment (PaulLeCam) [inline website/docs/guides/data-interactions/queries.mdx:7]: Missing file extension

File: website/docs/interact-with-data.mdx [hunk 25]
```diff
@@ -3,29 +3,31 @@ import TabItem from '@theme/TabItem'
 
 
 # Interact with data
-The final step of getting started with ComposeDB is interacting with your data. In this guide you will learn how to perform GraphQL queries and mutations using your composite.
+The final step of getting started with ComposeDB is interacting with your data using GraphQL. In this guide you will learn how to perform GraphQL queries and mutations using your composite.
 
-<!-- >💡 Want to interact with data from JavaScript? See [Client configuration](https://www.notion.so/Client-configuration-6147b53d9e01478f91bd81194c4a39c2). -->
+:::tip
+Want to interact with data using JavaScript instead? See [Client setup](./guides/composedb-client/javascript-client)
```
Comment (PaulLeCam) [inline website/docs/interact-with-data.mdx:9]: Missing file extension

File: website/docs/interact-with-data.mdx [hunk 26]
```diff
@@ -234,18 +236,20 @@ Although you can query records created by other accounts, you can only perform m
...
+Congratulations — you’re on your way to becoming a ComposeDB developer! 🔥 
 
 Visit [Next Steps](./next-steps.mdx) for more integration guides and opportunities to contribute to the ComposeDB on Ceramic ecosystem.
 
-<!-- ## Related Guides
+## Related Guides
 For more detailed descriptions and examples, see our advanced guides:
 
-- [Data interactions](https://www.notion.so/Data-interactions-249a50eba41341e8954511175f468fa2)
+- [Authentication for Mutations](./guides/composedb-client/user-sessions)
```
Comment (PaulLeCam) [inline website/docs/interact-with-data.mdx:247]: Missing file extension

File: website/docs/interact-with-data.mdx [hunk 27]
```diff
@@ -234,18 +236,20 @@ Although you can query records created by other accounts, you can only perform m
...
+
+- [Data Interactions](./guides/data-interactions)
 
-- [Mutations](https://www.notion.so/Mutations-0a93e6f4a004491c83e9908549f5522a)
+- [Queries](./guides/data-interactions/queries)
```
Comment (PaulLeCam) [inline website/docs/interact-with-data.mdx:251]: Missing file extension

File: website/docs/interact-with-data.mdx [hunk 28]
```diff
@@ -234,18 +236,20 @@ Although you can query records created by other accounts, you can only perform m
...
 Visit [Next Steps](./next-steps.mdx) for more integration guides and opportunities to contribute to the ComposeDB on Ceramic ecosystem.
 
-<!-- ## Related Guides
+## Related Guides
 For more detailed descriptions and examples, see our advanced guides:
 
-- [Data interactions](https://www.notion.so/Data-interactions-249a50eba41341e8954511175f468fa2)
+- [Authentication for Mutations](./guides/composedb-client/user-sessions)
+
+- [Data Interactions](./guides/data-interactions)
```
Comment (PaulLeCam) [inline website/docs/interact-with-data.mdx:249]: Missing file extension

File: website/docs/interact-with-data.mdx [hunk 29]
```diff
@@ -234,18 +236,20 @@ Although you can query records created by other accounts, you can only perform m
...
 
-- [Mutations](https://www.notion.so/Mutations-0a93e6f4a004491c83e9908549f5522a)
+- [Queries](./guides/data-interactions/queries)
 
-- [Queries](https://www.notion.so/Queries-4aba20cbf09d4f2685d9ab5412ce952f)
+- [Mutations](./guides/data-interactions/mutations)
```
Comment (PaulLeCam) [inline website/docs/interact-with-data.mdx:253]: Missing file extension

File: website/docs/interact-with-data.mdx [hunk 30]
```diff
@@ -234,18 +236,20 @@ Although you can query records created by other accounts, you can only perform m
...
 
-- [Client configuration](https://www.notion.so/Client-configuration-6147b53d9e01478f91bd81194c4a39c2): Perform queries and mutations from JavaScript -->
+- [ComposeDB Client setup](./guides/composedb-client/javascript-client)
```
Comment (PaulLeCam) [inline website/docs/interact-with-data.mdx:255]: Missing file extension

File: website/docs/introduction.mdx [hunk 31]
```diff
@@ -24,17 +24,18 @@ ComposeDB on Ceramic powers Web3 applications and use cases, such as:
 
 
 ## Getting Started
-Try the [Getting Started](./getting-started) guide to set up your environment, create your model, and interact with data using the ComposeDB CLI.
+Begin with the [**Getting Started →**](./getting-started) guide to set up your environment, create your model, and interact with data using the ComposeDB CLI.
```
Comment (PaulLeCam) [inline website/docs/introduction.mdx:27]: Missing file extension

File: website/docs/introduction.mdx [hunk 32]
```diff
@@ -24,17 +24,18 @@ ComposeDB on Ceramic powers Web3 applications and use cases, such as:
...
-### Integration Guides - Coming Soon
+## Go Deeper
+### Integration Guides
 After Getting Started, go deeper by jumping into our development guides:
 
-- [Data Modeling](https://www.notion.so/Data-Modeling-2599a8a3f4bd47d29c4898780a80f9cd)
-- [Application setup](https://www.notion.so/Application-setup-4ba61ae9757f4fbf9769aff4e95d1b93)
-- [User authentication](https://www.notion.so/User-authentication-298ad2839e02449caed1e39a9dd6b045)
-- [Data interactions](https://www.notion.so/Data-interactions-249a50eba41341e8954511175f468fa2) -->
+- [Data Modeling](./guides/data-modeling/)
```
Comment (PaulLeCam) [inline website/docs/introduction.mdx:34]: Missing file extension

File: website/docs/introduction.mdx [hunk 33]
```diff
@@ -24,17 +24,18 @@ ComposeDB on Ceramic powers Web3 applications and use cases, such as:
...
+- [Setting Up Your Client](./guides/composedb-client)
```
Comment (PaulLeCam) [inline website/docs/introduction.mdx:35]: Missing file extension

File: website/docs/introduction.mdx [hunk 34]
```diff
@@ -24,17 +24,18 @@ ComposeDB on Ceramic powers Web3 applications and use cases, such as:
...
+- [Authenticate Users](./guides/composedb-client/authenticate-users)
```
Comment (PaulLeCam) [inline website/docs/introduction.mdx:36]: Missing file extension

File: website/docs/introduction.mdx [hunk 35]
```diff
@@ -24,17 +24,18 @@ ComposeDB on Ceramic powers Web3 applications and use cases, such as:
...
+- [Data interactions](./guides/data-interactions)
```
Comment (PaulLeCam) [inline website/docs/introduction.mdx:37]: Missing file extension

File: website/docs/next-steps.mdx [hunk 36]
```diff
@@ -6,16 +6,13 @@ import TabItem from '@theme/TabItem'
 After learning the foundations of ComposeDB with the [Getting Started](./getting-started.mdx) guide, you are now ready to start integrating ComposeDB into your application and run in production.
 
 ## Integration Guides
-- Watch this space for more guides, including client and server configuration.
-<!-- To set up your application, first complete @Client configuration then @Server configuration. -->
+- Visit the [Guides](./guides) section to learn more about creating & interacting with data
```
Comment (PaulLeCam) [inline website/docs/next-steps.mdx:9]: Missing file extension

File: website/docs/next-steps.mdx [hunk 37]
```diff
@@ -6,16 +6,13 @@ import TabItem from '@theme/TabItem'
...
 - Create Ceramic App: [Blog](https://blog.ceramic.network/launching-create-ceramic-app/?ref=the-ceramic-blog-newsletter)
 
 ## Go Further
 - Join the [Community](./community.mdx)
 - Learn more about [Core Concepts](./core-concepts.mdx)
-
-<!-- 
-- Level up your [Data Modeling](https://www.notion.so/Data-Modeling-2599a8a3f4bd47d29c4898780a80f9cd)
-- Perform more advanced [Data interactions](https://www.notion.so/Data-interactions-249a50eba41341e8954511175f468fa2) -->
+- Level up your [Data Modeling](./guides/data-modeling)
```
Comment (PaulLeCam) [inline website/docs/next-steps.mdx:17]: Missing file extension

File: website/docs/next-steps.mdx [hunk 38]
```diff
@@ -6,16 +6,13 @@ import TabItem from '@theme/TabItem'
...
+- Perform more advanced [Data interactions](./guides/data-interactions)
```
Comment (PaulLeCam) [inline website/docs/next-steps.mdx:18]: Missing file extension

**Review by PaulLeCam — CHANGES_REQUESTED** [review]

File: website/docs/guides/composedb-client/user-sessions.mdx (hunk 22 above)
Comment (3avi) [reply website/docs/guides/composedb-client/user-sessions.mdx:207]: i didn't see those warnings in the DID docs, what should we put there?

Comment (PaulLeCam) [reply website/docs/guides/composedb-client/user-sessions.mdx:207]: The warning is on the page this example seem to be copied from, see https://did.js.org/docs/guides/using-with-composedb-client

File: website/docs/introduction.mdx [hunk 39]
```diff
@@ -24,18 +24,19 @@ ComposeDB on Ceramic powers Web3 applications and use cases, such as:
...
-### Integration Guides - Coming Soon
+## Go Deeper
+### Integration Guides
 After Getting Started, go deeper by jumping into our development guides:
 
-- [Data Modeling](https://www.notion.so/Data-Modeling-2599a8a3f4bd47d29c4898780a80f9cd)
-- [Application setup](https://www.notion.so/Application-setup-4ba61ae9757f4fbf9769aff4e95d1b93)
-- [User authentication](https://www.notion.so/User-authentication-298ad2839e02449caed1e39a9dd6b045)
-- [Data interactions](https://www.notion.so/Data-interactions-249a50eba41341e8954511175f468fa2) -->
+- [Data Modeling](./guides/data-modeling/data-modeling.mdx/)
```
Comment (PaulLeCam) [inline website/docs/introduction.mdx:34]:
```suggestion
- [Data Modeling](./guides/data-modeling/data-modeling.mdx)
```

**Review by PaulLeCam — CHANGES_REQUESTED** [review]
Thanks for the changes, I just noticed one extra trailing slash + warning about using local storage still missing.

**Review by PaulLeCam — DISMISSED** [review]
Thanks for the changes!

**Review by PaulLeCam — APPROVED** [review]

**Pull request merged**